src/Models/Classes/AdapterDataClasses/ADTInstanceTimeSeriesConnectionData.ts

```diff
@@ -0,0 +1,15 @@
+import { IAdapterData, IADXConnection } from '../../Constants/Interfaces';
+
+class ADTInstanceTimeSeriesConnectionData implements IAdapterData {
```

Comment:
nit: does it need to be TimeSeries specific? Could we call it `DataHistory` instead?

Reply:
Since the name of the method in Azure Management Rest api is /timeSeriesDatabaseConnections, I wanted to keep it aligned with it.